src/main/java/threads/FirstThread.java

```diff
@@ -0,0 +1,18 @@
+package threads;
+
+import model.DemoObject;
+
+public class FirstThread implements Runnable {
+    private DemoObject input;
+
+    public FirstThread(DemoObject input) {
+        this.input = input;
+    }
+
+    public void run() {
+        while (input.getCounter() < 100) {
+            input.setCounter(input.getCounter() + 1);
```

Comment:
- Incrementation of counter should be provided by count method of class DemoObject (btw, weird class name, why not Counter?)